lib/pages/detail/detail_content.dart

```diff
@@ -36,14 +37,15 @@ class DetailContent extends StatelessWidget {
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        if ((company.isFriend ?? false))
-          FriendsBar(),
+        if ((company.isFriend ?? false)) FriendsBar(),
         const SizedBox(height: 20.0),
         Padding(
           padding: const EdgeInsets.symmetric(horizontal: 17.0),
           child: Column(
             crossAxisAlignment: CrossAxisAlignment.start,
             children: [
+              _ScoreSection(company: company),
+
               Align(
```

Comment:
Musisz chyba usunąć od 49 do 144 linii. Najlepiej usuwaj po elemencie i sprawdzaj czy się kompiluje i jak wygląda. Jak ok to commit i powtarzasz do skutku. Jak coś idzie nie tak to zawsze możesz zrobić discard changes.